tests/foreman/ui/test_rhc.py

```diff
@@ -185,6 +185,6 @@ def test_positive_configure_cloud_connector(
 
     assert rhc_status.status == 0
     assert "Connected to Red Hat Subscription Management" in rhc_status.stdout
-    assert "The Red Hat connector daemon is active" in rhc_status.stdout
+    assert "The Remote Host Configuration daemon is active" in rhc_status.stdout
```

Comment:
@jameerpathan111 In the PRT results it seems, this change wasn't required for 6.12.z/6.11.z, can you check and update PR?

Reply:
@jameerpathan111 mentioned in 6.11.z PR that this change is happen due el8.8 and this old 6.12.z and 6.11.z templates are still on el8.7, so let's trust him and merge it as is